### PR TITLE
[solution] [minor] medium-promise-all

### PR DESCRIPTION
**Pull request by shalamai**
# Challenge [medium-promise-all]

## Checklist

- [ ] Make sure that you are opening the PR with a single solution (one solution per PR)
- [ ] The solution must have at least several references for the readers to dive in depth

### CODE REVIEW

**Review by bdvorianov — APPROVED** [review]

File: uk/medium-promise-all.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ declare function PromiseAll<T extends unknown[]>(values: readonly [...T]): Promi
 declare function PromiseAll<T extends unknown[]>(values: readonly [...T]): Promise<T extends Promise<infer R> ? R : T>
 ```
 
-Рішення досі неробоче, тому що `T`, не об'єднання, а кортеж.
+Рішення досі помилкове, тому що `T`, не об'єднання, а кортеж.
```
Comment (ghaiklor) [inline uk/medium-promise-all.md:63]: What is wrong with the dot? 🤔 

```suggestion
Рішення досі помилкове, тому що `T`, не об'єднання, а кортеж.
```

**Review by ghaiklor — APPROVED** [review]

File: uk/medium-promise-all.md (hunk 1 above)
Comment (shalamai) [reply uk/medium-promise-all.md:63]: image

Comment (ghaiklor) [reply uk/medium-promise-all.md:63]: Strange, really strange

Comment (ghaiklor) [reply uk/medium-promise-all.md:63]: Am I the only with the highlighted dot in red?

**Pull request merged**